bigquery_etl/cli/query.py

```diff
@@ -711,8 +723,10 @@ def backfill(
             "date_partition_parameter", "submission_date"
         )
         date_partition_offset = metadata.scheduling.get("date_partition_offset", 0)
-        scheduling_parameters = metadata.scheduling.get("parameters", [])
 
+        scheduling_parameters = scheduling_parameters_override
+        if not scheduling_parameters_override:
+            scheduling_parameters = metadata.scheduling.get("parameters", [])
```

Comment:
"Override" suggests it isn't the default:
```suggestion
        if scheduling_parameters_override:
            scheduling_parameters = scheduling_parameters_override
```
Can't suggest on a removal diff but this is without deleting the previous line 714